include/util.h

```diff
@@ -1,58 +1,66 @@
-
 #ifndef UTIL_H
- 
 #define UTIL_H
 
+#define clamp(value, min, max) \
```

Comment:
These 3 macros are not side-effect free, I'll need to transform them in static inline functions.